exporters/scss/src/js/generators/simple.ts

```diff
@@ -91,8 +91,8 @@ export function generateSimple(
     } else if (token.tokenType === 'Radius') {
       value = printUnit(token.value.radius.measure, token.value.radius.unit);
     } else if (token.tokenType === 'GenericToken') {
-      const unitLessProp = token.properties.find((prop) => prop.codeName === 'unitless');
-      if (unitLessProp && !!unitLessProp.value) {
+      const unitLessProp = token.propertyValues.unitless;
```

Comment:
I know I'm adding a comment to code that had already been existing from the past but is there any reason why a word `unitless` is camelCased? Is there a possibility of any chance to refactor this const so it gets propper naming?

Reply:
🤷 fixed